The old style of casting in the rcu_dereference fails to compile with
the newer rcu_dereference macro.

---
 security/apparmor/inline.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/security/apparmor/inline.h
+++ b/security/apparmor/inline.h
@@ -19,7 +19,7 @@ static inline int mediated_filesystem(st
 
 static inline struct aa_task_context *aa_task_context(struct task_struct *task)
 {
-	return rcu_dereference((struct aa_task_context *)task->security);
+	return (struct aa_task_context *) rcu_dereference(task->security);
 }
 
 /**
